django_freeradius/base/models.py

```diff
@@ -90,6 +90,12 @@
 )
 
 
+if app_settings.CUSTOM_TYPES_EXTRA:  # if cutom NAS types have been defined.
+    RAD_NAS_TYPES_ALL = RAD_NAS_TYPES + app_settings.CUSTOM_TYPES_EXTRA
+else:
+    RAD_NAS_TYPES_ALL = RAD_NAS_TYPES
```

Comment:
now after you deafult the new setting to an empty tuple you can avoid the `if`, as a consequence the coverage should not decrease.